.github/workflows/ci.generate.ts

```diff
@@ -700,6 +701,24 @@ const ci = {
             "df -h",
           ].join("\n"),
         },
+        {
```

Comment:
The binary is now smoke-tested (in sysroot and normally) after it boots to get faster feedback.

Reply:
Very cool, thanks!